src/Microsoft.Extensions.Logging.Testing/XunitLoggerProvider.cs

```diff
@@ -55,13 +55,23 @@ public XunitLogger(ITestOutputHelper output, string category, LogLevel minLogLev
                 return;
             }
             var firstLinePrefix = $"| {_category} {logLevel}: ";
-            var lines = formatter(state, exception).Split('\n');
-            WriteLine(firstLinePrefix + lines.First().TrimEnd(NewLineChars));
+            var lines = formatter(state, exception).Split(NewLineChars, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
You probably shouldn't remove empty entries here, it will collapse empty lines...